public/resources/css/layout/_grids.scss

```diff
@@ -389,6 +389,18 @@ limitations under the License.
     width: auto;
   }
 }
+
+@media handheld and (max-width: 480px), screen and (max-width: 480px), screen and (max-width: 480px) {
```

Comment:
On really small screens, like the iphone 5s, we get right of the left and right padding to give the text more space